manual_administrator/README_Openvpn.md

```diff
@@ -0,0 +1,116 @@
+#OpenVPN Router setup
+
+1. <a href="#Certificate">Certificate generation</a>
+2. <a href="#Zentyal">Zentyal Config</a>
+2. <a href="#DDWRT">DD WRT Config</a>
+
+### 1. Certificate generation <a id="Certificate"/> 
+- Open Zentyal Web Admin
+- Goto Certification Authority->General
+- Issue new Certificate with e.G. Common Name = cob4-X
+- Goto VPN->Servers->Download client bundle
+- Select Client Type and the newly generated client's certificate,
+  add the VPN Servers public IP as Server address (153.97.5.93)
+- Download the Certificate
+
+### 2. Zentyal Config <a id="Zentyal"/>
+- ssh to zentyal<br>
+  `ssh stud-admin@10.0.1.1`
+- cd directory to openvpn client config<br>
+  `/etc/openvpn/cob-kitchen-vpn.d/client-config.d`
+- create a new file with the excapt same name as the
+  generated client ceritivates common name<br>
+  `sudo vim <client cert name>`
+- add the following content:
+  "iroute <subnet_ip> <subnet_mask>"  without the quotes
+  for example: iroute 10.4.7.0 255.255.255.0
+- add new route to `/usr/share/zentyal/stubs/openvpn/openvpn.conf.mas`<br>
+  `route <subnet_ip> <subnet_mask>`<br>
+  e.g. `route 10.4.7.0 255.255.255.0`
+
+### 4. DD WRT Config <a id="DDWRT"/>
+open admin page and configure the follwing ddwrt settings:
+
+#### Setup->Basic Setup:
+- DNSMasq for DHCP = Enabled
+- DNSMasq for DNS = Enabled
+- DHCP-Authoritative = Enabled
+- NTP Client = Enabled 
+- NTP Client Server IP/Name = set to any NTP Server (eg. de.pool.ntp.org)
+
+#### Services->VPN:
+- Start OpenVPN Client = Enabled
+- Server IP/Name = <zentyals external ip>
```

Comment:
markdown language ignore the text between  "<" and ">" , could you maybe use inverted commas?